pyexchange/erisx.py

```diff
@@ -400,3 +403,23 @@ def parse_orders_list(messages: List[simplefix.FixMessage]) -> List:
             orders.append(order)
 
         return orders
+
+    @staticmethod
+    def parse_order_book(message: simplefix.FixMessage) -> List:
+        trade_count = int(message.get(268))
```

Comment:
Regarding nomenclature:  When I think of a level 2 _order book_, I think of two lists of _quotes_; a list of buys and a list of sells, each sorted by price.  The name `trade` implies an execution.

Reply:
Agreed, that makes more sense. Updated accordingly.